docs/source/algorithms/PolarizationCorrectionFredrikze-v1.rst

```diff
@@ -41,7 +57,67 @@ Entry in group  Measurement
 4               :math:`I_{aa}`
 ==============  ================
 
+Output from Polarized Neutron Reflectivity
+------------------------------------------
+
+The output of this algorithm is a :ref:`WorkspaceGroup <WorkspaceGroup>`, the resulting :ref:`WorkspaceGroup <WorkspaceGroup>` will have 2 entries, corresponding to the output specified spin state orders, if no specific output spin states are specified, it will be formatted as follows:
```

Comment:
```suggestion
The output of this algorithm is a :ref:`WorkspaceGroup <WorkspaceGroup>`. The resulting :ref:`WorkspaceGroup <WorkspaceGroup>` will have 2 entries, corresponding to the specified output spin state orders. If no specific output spin states are specified, it will be formatted as follows:
```